files/usr/lib/jeos-firstboot

```diff
@@ -26,7 +26,9 @@ TEXTDOMAIN='jeos-firstboot'
 . /etc/os-release
 . "$0-functions"
 
-EULA_FILE=/etc/YaST2/licenses/base/license.txt
+# Starting with SLE-15-SP1 that is the EULA file
+EULA_FILE=/usr/share/licenses/product/SLES/license.txt
```

Comment:
Doesn't look correct to me - what about other products (SLED, etc.) and openSUSE?